LICENSE.md

```diff
@@ -1,17 +1,17 @@
-Copyright © 2008 by Yii Software (https://www.yiiframework.com/)
+Copyright © 2008-present by Yii Software (<https://www.yiiframework.com/>)
```

Comment:
Year in copyright is the first year when it was used. No need for `-present`. I'd avoid changing LICENSE.md at all.